java-security-test/src/main/java/com/sap/cloud/security/test/JwtGenerator.java

```diff
@@ -215,6 +215,28 @@ public JwtGenerator withClaimValue(String claimName, JsonObject object) {
 		return this;
 	}
 
+	/**
+	 * Sets the claim with the given name to the given string value. Note: for
+	 * overwriting client Id claim, "azp" claim value should be overwritten instead
+	 * of deprecated "cid"
+	 *
+	 * @param claimName
+	 *            the name of the claim to be set.
+	 * @param object
+	 *            the JSONObject representation of claim value to be set.
+	 * @return the builder object.
+	 * @throws JsonParsingException
+	 *             if the given object does not contain valid json.
+	 */
+	public JwtGenerator withClaimValue(String claimName, JSONObject object) {
```

Comment:
why needed, there is already: public JwtGenerator withClaimValue(String claimName, JsonObject object)

Reply:
idea: change parameter to map.